base/dict.jl

```diff
@@ -189,6 +189,16 @@ function grow_to!{K,V}(dest::Associative{K,V}, itr, st)
     return dest
 end
 
+
+"""
+    similar(d::Dict, [element_type=Pair{K,V}])
+
+Create an empty dictionary with key type `K` and value type `V`. The second element
```

Comment:
Yeah, the docs have been written at many different times, so they are not all consistent with the rules established recently. See in particular point 4. at http://docs.julialang.org/en/latest/manual/documentation.html#Documentation-1.